web/app/components/app/configuration/debug/index.tsx

```diff
@@ -136,19 +136,19 @@ const Debug: FC<IDebug> = ({
 
   const { notify } = useContext(ToastContext)
   const logError = useCallback((message: string) => {
-    notify({ type: 'error', message, duration: 3000 })
+    notify({ type: 'error', message, duration: 6000 })
```

Comment:
I suggest we could add a default value here, and remove all those pre-defined numbers, so that we do not need to edit those files later, since they are the same.
https://github.com/langgenius/dify/blob/eff280f3e79cf1f75d3b11338def71832905a8aa/web/app/components/base/toast/index.tsx#L120

Reply:
Actually, the notify component already has a default value, with the duration set to 3000 by default. 
https://github.com/langgenius/dify/blob/eff280f3e79cf1f75d3b11338def71832905a8aa/web/app/components/base/toast/index.tsx#L92
However, I am not sure why most of the error messages specifically set it to 3000. On the 'Password Reset' page, the duration of notify is set to 5000.
https://github.com/langgenius/dify/blob/eff280f3e79cf1f75d3b11338def71832905a8aa/web/app/init/InitPasswordPopup.tsx#L35

Reply:
I guess it's because someone reported it before and only make that notification longer.

Reply:
Perhaps we could consider changing the default value to 6000 and removing all the predefined numbers.